docs/programming/MPI-C.md

```diff
@@ -8,10 +8,8 @@ Intel C++ Compiler, GCC, IntelMPI, and OpenMPI to create a
 multiprocessor ‘hello world’ program in C++.  This tutorial assumes
 the user has experience in both the Linux terminal and C++.
 
-__Resources:__
-- [http://www.dartmouth.edu/~rc/classes/intro_mpi/intro_mpi_overview.html](http://www.dartmouth.edu/~rc/classes/intro_mpi/intro_mpi_overview.html)  
+__Resource:__
```

Comment:
Like in other pages, could you replace different options or examples with tabs? For example: 
![Screenshot 2024-09-18 at 4 39 28 PM](https://github.com/user-attachments/assets/caff4e1d-db3a-45a0-b2ba-7391c96059b7)

Reply:
Also, something link this:
![Screenshot 2024-09-18 at 4 41 45 PM](https://github.com/user-attachments/assets/f2233d45-e13b-46ea-a21e-37a04e612d0a)